python/gps/algorithm/config.py

```diff
@@ -44,3 +41,12 @@
     'exp_step_upper': 0.5,
     'exp_step_lower': 1.0,
 }
+
+# AlgorithmMD
+ALG_MDGPS = {
+    'inner_iterations': 4,
+    'init_pol_wt': 0.01,
```

Comment:
Actually, I still need 'init_pol_wt', because the pol_info constructor requires it (and the policy_opt code uses pol_wt in several places). It doesn't matter for mdgps since the pol_wt remains constant, but it would be a good thing to separate out when refactor.